_i18n/ru/changes.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Все последние изменения в QMK"
+layout: qmk-title
+permalink: /ru/changes/
+---
+
+{% for change in site.changes %}
```

Comment:
```suggestion
{% for change in site.changes reversed %}
```
This way the most recent change is at the top of the page. See #78

Reply:
@kuchosauronad0 Thanks for letting know! Will commit this after merging the upstream PR.

Reply:
I just merged PR 78, this can go in now.

Reply:
@noroadsleft Thanks, done!